### DISCUSSION TITLE
Get typescript types from sanity v3 schemas

### DISCUSSION THREAD

**Discussion by emilmoberg** [sanity-io/sanity#4588] [Sanity Studio Support]
Hi! I do not understand from documentation how to get typescript types from schemas from v3, cause apparently something has been done with this version to make that easier?
With v2 I used sanity-codegen which worked kindof fine

**Comment by dogfrogfog** [comment]
try these
```
    "@sanity-typed/client": "^1.0.4",
    "@sanity-typed/groq": "^1.0.4",
    "@sanity-typed/types": "^4.0.4",
```
